### ISSUE TITLE
vm fails to start (macOS Intel)

### ISSUE THREAD

**Issue by ardeguire** [abiosoft/colima#343]
### Describe the Issue
VM hangs at start:
```
% colima start
INFO[0000] starting colima                              
INFO[0000] runtime: docker                              
INFO[0000] preparing network ...                         context=vm
INFO[0000] starting ...                                  context=vm
> msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
> msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
> msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
> msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
> msg="[hostagent] Waiting for the essential requirement 1 of 5: \"ssh\""
> msg="did not receive an event with the \"running\" status"
FATA[0600] error starting vm: error at 'starting': exit status 1 
```

#### Colima Version:

What is the output of `colima version`:
```
% colima version
colima version 0.4.2
git commit: f112f336d05926d62eb6134ee3d00f206560493b
```

#### Lima Version:

What is the output of `limactl --version`:
```
% limactl --version
limactl version 0.11.1
```

#### Qemu Version

What is the output of `qemu-img --version`:
```
% qemu-img --version
qemu-img version 7.0.0
```

### Operating System

- [ x] macOS Intel
- [ ] macOS m1
- [ ] Linux

### To Reproduce

1. `brew install colima`
2. `brew install docker`
3. `colima start`

### Expected behavior

VM to start

### Additional context

[ha.stderr.log](https://github.com/abiosoft/colima/files/8947603/ha.stderr.log)
[serial.log](https://github.com/abiosoft/colima/files/8947604/serial.log)

**Comment by ardeguire** [comment]
macOS Monterey 12.3.1 (21E258)
qemu was installed as a dependency with `brew install colima`

**Comment by ardeguire** [comment]
I don't know if this is relevant at all, but I am able to start lima with the "docker" configuration. It's kind of slow (~5 minutes), but it does eventually complete successfully complete

**Comment by ardeguire** [comment]
2019 MBP, 2.6 GHz i7.

My internet *is* relatively slow due to my company's proxy settings, but it should still work quickly once things are cached on my machine

**Comment by ardeguire** [comment]
I will close this issue for now, as I don't have the time to troubleshoot or power to change my company's proxy settings and other restrictions.

**Comment by abiosoft** [comment]
> I will close this issue for now, as I don't have the time to troubleshoot or power to change my company's proxy settings and other restrictions.

I would argue otherwise. Let us leave it open.

**Comment by fortesdev** [comment]
I am experiencing the exact same problem, Intel MacBook Pro (16-inch, 2019). It was working weeks ago, all of the sudden it broke. Cleaning $HOME/.lima/colima/ or deleting Colima did not help.

**Comment by abiosoft** [comment]
@fortesdev did you update anything?

**Comment by fortesdev** [comment]
Yes, @abiosoft, I updated the common:
Xcode to Version 13.4.1 + command line tools
And MacOS Monterrey to 12.4
But note this: I just deleted HEAD, installed v0.3.4, and that old version works like a charm.

**Comment by abiosoft** [comment]
@ardeguire did you restart your machine?

**Comment by ardeguire** [comment]
Yes, but I'm pretty sure I also restarted several times when I was attempting to troubleshoot when I first encountered the issue

**Comment by SmartManoj** [comment]
Duplicate of #215?
